src/components/context/QuizContext.tsx

```diff
@@ -51,11 +52,16 @@ export const QuizProvider = ({ children }: Props) => {
   const { mutateAsync: patchAppSettingAsync } = mutations.usePatchAppSetting();
   // current question idx
   // -1 if we are adding a new question
-  const [currentIdx, setCurrentIdx] = useState(0);
+  const [currentIdx, setCurrentIdx] = useState(-1);
 
   const [orderSetting, setOrderSetting] = useState<AppSetting>();
   const [order, setOrder] = useState<string[]>([]);
 
+  // This state indicates if the questions were received and the question order set correctly.
+  // It allows QuizNavigation to display the Add question button when the loading is terminated
```

Comment:
```suggestion
  // It allows QuizNavigation to display the Add question button when loading stops
```

Reply:
not quite sure of the english, but terminate sounds more like "résilier" than "finir"